src/Options.php

```diff
@@ -58,6 +68,13 @@ final class Options
     public function __construct(array $options = [])
     {
         $this->resolver = new OptionsResolver();
+        $this->defaultIntegrations = [
+            new ExceptionListenerIntegration(),
+            new ErrorListenerIntegration(null, false),
+            new FatalErrorListenerIntegration(),
+            new RequestIntegration(),
+            new TransactionIntegration(),
+        ];
```

Comment:
It's probably a micro optimization but shouldn't we move this to a private getter so they're only created (I know most constructors are not doing anything but still) when needed?